massa-storage/src/operation_indexes.rs

```diff
@@ -23,20 +23,21 @@ impl OperationIndexes {
     /// Arguments:
     /// * `operation`: the operation to insert
     pub(crate) fn insert(&mut self, operation: SecureShareOperation) {
-        if let Ok(o) = self
-            .operations
-            .try_insert(operation.id, Box::new(operation))
-        {
+        if !self.operations.contains_key(&operation.id) {
```

Comment:
see previous case of this, to avoid double hashing/lookup

Reply:
Done